src/views/orb-view.ts

```diff
@@ -26,8 +32,20 @@ import { isBoolean } from '../utils/type.utils';
 export interface IGraphInteractionSettings {
   isDragEnabled: boolean;
   isZoomEnabled: boolean;
+  keyboard: {
+    isEnabled: boolean;
+    zoomInFactor: number;
+    zoomOutFactor: number;
+    panFactor: number;
+    transitionMs: number;
+  };
 }
 
+const DEFAULT_ZOOM_IN_FACTOR = 1.2;
+const DEFAULT_ZOOM_OUT_FACTOR = 0.8;
+const DEFAULT_PAN_FACTOR = 25;
+const DEFAULT_TRANSITION_MS = 200;
```

Comment:
Maybe the default can be `0` - no transition.

Reply:
Done, Please check